src/components/Proposal/Status/index.tsx

```diff
@@ -27,23 +27,26 @@ const Status = () => {
     ? proposal.submittedTime.plus(scheme.maxSecondsForExecution)
     : bnum(0);
 
-  const executeProposal = function () {
-    daoService.execute(proposalId);
+  const executePendingAction = function (pendingAction) {
+    switch (pendingAction) {
+      case 4:
+        daoService.redeemContributionReward(
+          networkContracts.daostack.contributionRewardRedeemer,
+          scheme.address,
+          scheme.votingMachine,
+          proposalId,
+          proposal.to[0]

```

Comment:
Think this is incorrect, Dave seems to think it should be current account being sent here, not the `to` address which is just the controller